src/sentry/__init__.py

```diff
@@ -52,6 +59,9 @@ def is_docker():
 __build__ = get_revision()
 __docformat__ = "restructuredtext en"
 
+_build_or_empty = "" if __build__ is None else f"+{get_short_revision(__build__)}"
+__full_version__ = f"{__version__}{_build_or_empty}"
```

Comment:
You can just reduce this to `__full_version__ = __version__ if __build__ is None else f"{__version__}+{__build__[:12]}"`.

Also I don't think `__full_version__`  is a good name, it's confusing with the short sha. Maybe `__release_version__` and include a little comment about what it's for.

Reply:
Oh or `__semantic_version__`.

Reply:
Ehh, I see get_short_revision being used in https://github.com/getsentry/getsentry/pull/6265 and I kind of think `__full_version__` is ok now.

Reply:
I think release/semantic might be better. Will think about it tomorrow and add comments.